scripts/db_migrator.py

```diff
@@ -1056,9 +1072,31 @@ def version_4_0_3(self):
     def version_4_0_4(self):
         """
         Version 4_0_4.
-        This is the latest version for master branch
         """
         log.log_info('Handling version_4_0_4')
+
+        sflow_tbl = self.configDB.get_table('SFLOW')
+        for k, v in sflow_tbl.items():
```

Comment:
Please move this to migrate_sflow function.

Reply:
Fixed. Pls review